.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Running GoFish linter
+
+on:
+  pull_request:
+     branches:
+      - master
+     paths:
+      - '**.lua'
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+    - name: Setup Action
+      uses: actions/checkout@v2
+    - name: Install gofish
+      run: |
+        npm install -yq curl git sudo
```

Comment:
npm install? Shouldn't this be `apt install`?

https://github.com/fishworks/fish-food/blob/97f996c676eb7ba784c376bcf2883640bab691f5/.circleci/install-fish.sh#L9

Reply:
It is actually no even needed. The github actions runner comes with all needed binaries.
I have removed the line.

See a test run for the new version here https://github.com/fmotrifork/fish-food/runs/779285574?check_suite_focus=true